internal/test/server/doc.go

```diff
@@ -1,4 +1,4 @@
 package server
 
 //go:generate go run github.com/discord-gophers/goapi-gen --generate=types,server --package=server -o server.gen.go ../test-schema.yaml
-//go:generate go run github.com/matryer/moq -out server_moq.gen.go . ServerInterface
+//go:generate go run github.com/matryer/moq@latest -out server_moq.gen.go . ServerInterface
```

Comment:
I'm not sure why I can't go generate without that, but I need @latest for it to pass.
If it works on your end I could remove it, or even tag another version